### PR TITLE
Comparison with Mathematica

### PR DESCRIPTION
**Pull request by Upabjojr**

### CODE REVIEW

File: comparison_with_mma.tex [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Unlike SymPy, Mathematica's expressions are mutable,
+that is one can change parts of the expression tree without the need of
+creating a new object.
+The reactivity of Mathematica allows for a lazy updating of any references
+to that data structure.
+
+% Products and commutativity
+
+Product in Mathematica are determined by some builtin node types,
```
Comment (shivamvats) [inline comparison_with_mma.tex]: *Products

File: comparison_with_mma.tex [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+such as \texttt{Times}, \texttt{Dot}, and others.
+\texttt{Times} is overloaded by the * operator,
+and is always meant to represent a commutative operator.
+The other notable product is \texttt{Dot}, overloaded by the . operator.
+This products represents matrix multiplication,
```
Comment (shivamvats) [inline comparison_with_mma.tex]: *product

**Comment by shivamvats** [comment]
Could you please resolve the conflicts?

**Comment by aktech** [comment]
@Upabjojr Any references to the text?

**Comment by Upabjojr** [comment]
> Any references to the text?

Mathematica's documentation could be one. Prof. Fateman's review could be another one.

**Comment by moorepants** [comment]
This looks fine for merging as a first draft. It may need to be shortened once we see the whole length of the paper and other sections together.

**Comment by moorepants** [comment]
I'll merge later today unless there are objections.

**Pull request merged**